src/ice-breaker/about.md

```diff
@@ -0,0 +1,34 @@
+# ICE-breakers
+
+The **ICE-breaker groups** are an easy way to help out with rustc in a
+"piece-meal" fashion, without committing to a larger project.
+ICE-breaker groups are **[easy to join](#join)** (just submit a PR!)
+and joining does not entail any particular commitment.
+
+Once you [join an ICE ICE-breaker group](#join), you will be added to
+a list that receives pings on github whenever a new issue is found
+that fits the ICE-breaker group's criteria. If you are interested, you
+can then [claim the issue] and start working on it.
+
+Of course, you don't have to wait for new issues to be tagged! If you
+prefer, you can use the Github label for an ICE-breaker group to
+search for existing issues that haven't been claimed yet.
+
+## What issues are a good fit for ICE-breaker groups?
+
+"ICE-breaker issues" are intended to be **isolated** bugs of **middle
+priority**:
+
+- By **isolated**, we mean that we do not expect large-scale refactoring
+  to be required to fix the bug.
+- By **middle priority**, we mean that we'd like to see the bug fixed,
+  but it's not such a burning problem that we are dropping everything
+  else to fix it. The danger with such bugs, of course, is that they
+  can accumulate over time, and the role of the ICE-breaker groups is
+  to try and stop that from happening!
+
+<a name="join"></a>
```

Comment:
What was the idea with this?

Reply:
that we are going to want to link to this page, I think

Reply:
I'm questioning mainly because I wanted to know if you wanted to merge this or was WIP.

Reply:
in particular I wanted a "stable" anchor in case we change the section heading later